Subject: remove references to removed brp scripts

Some brp-scripts are not used in openSUSE, remove references
to it

--- ./platform.in.orig	2018-10-16 09:22:16.004374524 +0000
+++ ./platform.in	2018-10-16 09:24:58.891939175 +0000
@@ -80,8 +80,6 @@
 %__os_install_post    \
     %{?__brp_compress} \
     %{?__brp_strip} \
-    %{?__brp_strip_static_archive} \
-    %{?__brp_strip_comment_note} \
 %{nil}
 
 %__spec_install_post\
